File: java/program/Shirt.java

public class Shirt extends Clothing {
    protected String color;
    protected String sleeveType;

    // Konstruktor
    public Shirt(int idProduct, String name, String brand, double price, String size, String material, String gender, String color, String sleeveType) {
        super(idProduct, name, brand, price, size, material, gender);
        this.color = color;
        this.sleeveType = sleeveType;
    }

    public String getColor() {
        return color;
    }

    public String getSleeveType() {
        return sleeveType;
    }
}

File: php/program/Shirt.php
<?php

//IMPORT FILE
require('Clothing.php');

//KELAS SHIRT ADALAH CHILD DARI CLOTHING
class Shirt extends Clothing
{
    //PRIVATE ATRIBUT
    private $color = '';
    private $sleeveType = '';

    //KONSTRUKTOR
    public function __construct($id_product = '', $name = '', $brand = '', $price = '', $size = '', $material = '', $gender = '', $color = '', $sleeveType = '')
    {
        parent::__construct($id_product, $name, $brand, $price, $size, $material, $gender);
        $this->color = $color;
        $this->sleeveType = $sleeveType;
    }

    //GETTER DAN SETTER COLOR
    public function set_color($color)
    {
        $this->color = $color;
    }

    public function get_color()
    {
        return $this->color;
    }

    //GETTER DAN SETTER SLEEVE TYPE
    public function set_sleeveType($sleeveType)
    {
        $this->sleeveType = $sleeveType;
    }

    public function get_sleeveType()
    {
        return $this->sleeveType;
    }

}

?>
